cron-whitelist.json

```diff
@@ -80,5 +80,45 @@
 				}
 			}
 		}
+	},
+	"nextcloud": {
+		"audits": {
+			"bsc#1162254": {
+				"comment": "default-disabled cron job that runs PHP cleanup logic for nextcloud as unprivileged user",
+				"digests": {
+					"/etc/cron.d/nextcloud": "sha256:176aed8f9adc79cc3d0a75a223730d621cf9435d7baee03823afc558c110b76d"
+				}
+			}
+		}
+	},
+	"texlive-filesystem": {
+		"audits": {
+			"bsc#1150556": {
+				"comment": "more or less broken security concept, disagreement with package maintainer. due to importance of TeX a wildcard whitelisting until the matter is settled",
+				"digests": {
+					"/etc/cron.daily/suse-texlive": "skip:<none>"
```

Comment:
I worry a bit that we might forget to change this to a fixed hash once this is finally resolved. Do you see any better strategy than 'try not to forget'?

Reply:
We could open an issue here on GitHub. We also have the cron job tracker bug that I won't close before all cron jobs are sane. Otherwise I don't see any compelling technical solution.

Reply:
True. I'm merging this, can you please open the issue?